src/database/entities/user.entity.ts

```diff
@@ -0,0 +1,106 @@
+import { Entity, PrimaryGeneratedColumn, Column } from 'typeorm';
+
+@Entity('auth.users')
```

Comment:
o nome da tabela é `auth.users` ou `auth` é o schema do db e `users` a tabela?

Reply:
CREATE TABLE
  auth.users (
    instance_id uuid NULL,
    id uuid NOT NULL,
    aud character varying(255) NULL,
    role character varying(255) NULL,
    email character varying(255) NULL,
    encrypted_password character varying(255) NULL,
    email_confirmed_at timestamp with time zone NULL,
    invited_at timestamp with time zone NULL,
    confirmation_token character varying(255) NULL,
    confirmation_sent_at timestamp with time zone NULL,
    recovery_token character varying(255) NULL,
    recovery_sent_at timestamp with time zone NULL,
    email_change_token_new character varying(255) NULL,
    email_change character varying(255) NULL,
    email_change_sent_at timestamp with time zone NULL,
    last_sign_in_at timestamp with time zone NULL,
    raw_app_meta_data jsonb NULL,
    raw_user_meta_data jsonb NULL,
    is_super_admin boolean NULL,
    created_at timestamp with time zone NULL,
    updated_at timestamp with time zone NULL,
    phone text NULL DEFAULT NULL::character varying,
    phone_confirmed_at timestamp with time zone NULL,
    phone_change text NULL DEFAULT ''::character varying,
    phone_change_token character varying(255) NULL DEFAULT ''::character varying,
    phone_change_sent_at timestamp with time zone NULL,
    confirmed_at timestamp with time zone NULL DEFAULT LEAST(email_confirmed_at, phone_confirmed_at),
    email_change_token_current character varying(255) NULL DEFAULT ''::character varying,
    email_change_confirm_status smallint NULL DEFAULT 0,
    banned_until timestamp with time zone NULL,
    reauthentication_token character varying(255) NULL DEFAULT ''::character varying,
    reauthentication_sent_at timestamp with time zone NULL,
    is_sso_user boolean NOT NULL DEFAULT false,
    deleted_at timestamp with time zone NULL
  );

ALTER TABLE
  auth.users
ADD
  CONSTRAINT users_pkey PRIMARY KEY (id)